apps/api/src/domain/generated/coordinator/dto/connect-coordinator.dto.ts

```diff
@@ -1,4 +1,6 @@
-export class ConnectCoordinatorDto {
-  id?: string
-  personId?: string
-}
+
```

Comment:
Please run `yarn format` prior your commits so no other auto generated code is affected

Reply:
Done.